src/components/PublicRoadmap.astro

```diff
@@ -1,16 +1,16 @@
 ---
+import { Badge, Icon } from '@astrojs/starlight/components';
```

Comment:
I am not a big fan of the styling of the `Badge` component that comes with starlight, but I suppose it's ok for now